repository_service_tuf/cli/admin/__init__.py

```diff
@@ -12,12 +12,32 @@
 
 from repository_service_tuf.cli import click, rstuf
 
+HEADERS_EXAMPLE = (
+    "'Authorization: Bearer <token>, Content-Type: application/json'"
+)
+
 
-def _set_settings(context: click.Context, api_server: Optional[str]):
+def _set_settings(
+    context: click.Context, api_server: Optional[str], headers: Optional[str]
+):
     """Set context.obj['settings'] attributes."""
     settings = context.obj["settings"]
     if api_server:
         settings.SERVER = api_server
+    if headers:
+        try:
+            settings.HEADERS = dict(
```

Comment:
We can certainly improve input validation here ([issue filed](https://github.com/repository-service-tuf/repository-service-tuf-cli/issues/753)), but I think this works for now, especially if we are careful on the server-side. Thanks!